packages/css/test/instance/__snapshots__/inline.test.js.snap

```diff
@@ -91,52 +91,6 @@ exports[`hydration only inserts rules that are not in the critical css 3`] = `
   box-shadow: -15px -15px 0 0 aqua,-30px -30px 0 0 cornflowerblue;
 }
 
-@font-face {
```

Comment:
those were duplicated in the snapshot - I'm not entirely sure why, but the test setup here (and in other 3 tests that very similar) was a little bit convoluted, with using JSDOM manually etc. I've migrated those tests to `disableBrowserEnvTemporarily` strategy and this has "fixed" the problem accidentally.